client/src/webview-js/cnxml-preview/content-common.less

```diff
@@ -114,7 +114,7 @@ figure:not([data-orient="vertical"]) {
   }
 
   > figcaption {
-    display: table-caption;
+    // display: table-caption;
```

Comment:
Do you mean to have this comment here? I tend to shy away from keeping commented code around but of course, there are certain conditions when we can make exceptions.

Reply:
Yeah -- I left it as a comment thinking just in case the file is copied again [from REX](https://github.com/openstax/rex-web/tree/master/generic-styles/) it might be somewhat clearer it was a change we made vs a diff introduced by changes in that repo 🤷‍♂️ (or at least trigger someone to do a `git blame` on the line in question).

Reply:
It's probably fine then.